runtime/src/main/java/io/quarkiverse/tekton/client/TektonClientProducer.java

```diff
@@ -0,0 +1,20 @@
+package io.quarkiverse.tekton.client;
+
+import javax.enterprise.inject.Produces;
+import javax.inject.Singleton;
+
+import io.fabric8.kubernetes.client.Config;
+import io.fabric8.tekton.client.DefaultTektonClient;
+import io.fabric8.tekton.client.TektonClient;
+import io.quarkus.arc.DefaultBean;
+
+@Singleton
+public class TektonClientProducer {
+
+    @DefaultBean
+    @Singleton
+    @Produces
+    public TektonClient tektonClient(Config config) {
```

Comment:
Hmmm, another question, where is this `Config` object coming from?

Reply:
It's from kubernetes-client extension: https://github.com/quarkusio/quarkus/blob/bce536fe09d3902aaee1a23cbca17939af98f9b0/extensions/kubernetes-client/runtime/src/main/java/io/quarkus/kubernetes/client/runtime/KubernetesClientProducer.java#L21 (which is a dependency for this extension).